frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -585,6 +581,11 @@ export const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTablePro
                         '-'
                       )}
                     </td>
+                    {releaseNotes && (
+                      <td className="hidden-xs hidden-sm">
+                        <ReleaseNotesLink channel={channel} version={update.version} />
```

Comment:
The slightly weird thing is that the links in this table could appear / disappear when changing the channel. I'm not sure what to do about that, though. It would probably be rare in practice.

I wonder if we really need the channel checking since we already check the semver prerelease value. Is there ever a time where the link wouldn't work for a version that doesn't have a semver prerelease?

Reply:
> The slightly weird thing is that the links in this table could appear / disappear when changing the channel. I'm not sure what to do about that, though. It would probably be rare in practice.

Indeed.  And that's what happens in the cluster settings box at the top of the page based on previous discussion.  Do we need to revisit?

Reply:
I commented on the JIRA issue. I think we should consider dropping the check, but we can look at it separately.